Wireframe/index.html

```diff
@@ -8,25 +8,37 @@
   </head>
   <body>
     <header>
-      <h1>Wireframe</h1>
-      <p>
-        This is the default, provided code and no changes have been made yet.
+      <h1>Wireframes, README files and Git Branches</h1>
+      <p>In this website your will learn what they are, how to use them and why they are helpful
       </p>
     </header>
     <main>
       <article>
-        <img src="placeholder.svg" alt="" />
-        <h2>Title</h2>
+        <img src="https://media.geeksforgeeks.org/wp-content/uploads/20240702120959/Readme1.png" alt="An image of a README.md file that reads 'README'" />
+        <h2>What is the purpose of a README file?</h2>
         <p>
-          Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam,
-          voluptates. Quisquam, voluptates.
+          README Files are a common way to document the contents and structure of a folder and/or a dataset so that a researcher can locate the information they need. Data documentation can be maintained in a variety of forms. Explore additional Documentation & Metadata practices.
         </p>
-        <a href="">Read more</a>
+        <a href="https://datamanagement.hms.harvard.edu/collect-analyze/documentation-metadata/readme-files#:~:text=README%20Files%20are%20a%20common,Explore%20additional%20Documentation%20%26%20Metadata%20practices.">Read more</a>
```

Comment:
I don't think you need the part after # in this link. It works for me this way:

https://datamanagement.hms.harvard.edu/collect-analyze/documentation-metadata/readme-files

Reply:
Same suggestion for other links in you html.